azure-iot-device/azure/iot/device/iothub/abstract_clients.py

```diff
@@ -362,6 +377,16 @@ def create_from_symmetric_key(cls, symmetric_key, hostname, device_id, **kwargs)
     def receive_message(self):
         pass
 
+    @abc.abstractmethod
```

Comment:
Unrelated to this PR, but these abstract definitions were missing, so I added them